content/authorization/guides/system-vendor/system-user/client delegation/_index.nb.md

```
@@ -1,38 +1,28 @@
-Klientdelegerings-API-et tilbyr metoder for å administrere klienter for en agentsystembruker fra et tredjepartssystem.
 
-Guide til å gjøre dette som sluttbruker: [Klientdelegering](/nb/authorization/guides/end-user/system-user/delegate-clients/)
+## 1 Hent systembrukere for klientsystem
 
-## 1 List alle agent-systembrukere for organisasjon
+Finn alle systembrukere for klientsystem som er knyttet til din organisasjon.
 
-Returnerer en liste over alle agentbrukere i systemet som er knyttet til organisasjon
 
-### Endepunkt
+#### API endepunkt: GET authentication/api/v1/enduser/systemuser/agents
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)